cli/tests/data/package_build/cassandra/marathon.json.mustache

```diff
@@ -0,0 +1,174 @@
+
```

Comment:
why are the cassandra example files so complicated? we should use simpler examples